<ui version="4.0" >
 <class>QgsPointDialogBase</class>
 <widget class="QDialog" name="QgsPointDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Reference points</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" >
     <item>
      <widget class="QFrame" name="frame4" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="margin" >
         <number>11</number>
        </property>
        <item>
         <widget class="QToolButton" name="tbnZoomIn" >
          <property name="toolTip" >
           <string>Zoom in</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff/>
           </iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbnZoomOut" >
          <property name="toolTip" >
           <string>Zoom out</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff/>
           </iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbnZoomToLayer" >
          <property name="toolTip" >
           <string>Zoom to the raster extents</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff/>
           </iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbnPan" >
          <property name="toolTip" >
           <string>Pan</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff/>
           </iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame5" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>0</number>
        </property>
        <property name="margin" >
         <number>11</number>
        </property>
        <item>
         <widget class="QToolButton" name="tbnAddPoint" >
          <property name="toolTip" >
           <string>Add points</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff/>
           </iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbnDeletePoint" >
          <property name="enabled" >
           <bool>true</bool>
          </property>
          <property name="toolTip" >
           <string>Delete points</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset>
            <normaloff/>
           </iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="canvasFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="textLabel1_2" >
         <property name="text" >
          <string>Raster file:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3" >
        <widget class="QLineEdit" name="leSelectRaster" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="0" column="4" >
        <widget class="QPushButton" name="pbnSelectRaster" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="textLabel2" >
         <property name="text" >
          <string>Transform type:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4" >
        <widget class="QComboBox" name="cmbTransformType" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="lblSelectModifiedRaster" >
         <property name="text" >
          <string>Modified raster:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3" >
        <widget class="QLineEdit" name="leSelectModifiedRaster" />
       </item>
       <item row="2" column="4" >
        <widget class="QPushButton" name="pbnSelectModifiedRaster" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>World file:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3" >
        <widget class="QLineEdit" name="leSelectWorldFile" />
       </item>
       <item row="3" column="4" >
        <widget class="QPushButton" name="pbnSelectWorldFile" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QPushButton" name="pbnGenerateWorldFile" >
         <property name="text" >
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QPushButton" name="pbnGenerateAndLoad" >
         <property name="text" >
          <string>Create and load layer</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4" >
        <widget class="QPushButton" name="pbnClose" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3" >
        <widget class="QPushButton" name="pbnSaveGCPs" >
         <property name="text" >
          <string>Save GCPs</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <widget class="QPushButton" name="pbnLoadGCPs" >
         <property name="text" >
          <string>Load GCPs</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>18</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="georeferencer.qrc" />
 </resources>
 <connections/>
</ui>
